.devcontainer/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Find the Dockerfile for mcr.microsoft.com/azure-functions/dotnet:3.0-dotnet3-core-tools at this URL
+# https://github.com/Azure/azure-functions-docker/blob/master/host/3.0/buster/amd64/dotnet/dotnet-core-tools.Dockerfile
+FROM mcr.microsoft.com/azure-functions/dotnet:3.0-dotnet3-core-tools
```

Comment:
Please could you update to use the dotnet SDK 5 base image:
```
FROM mcr.microsoft.com/dotnet/sdk:5.0
```

Reply:
Thanks, sorted.